Sources/Widgets/Manipulators/LineManipulator/index.js

```diff
@@ -41,15 +41,16 @@ function vtkLineManipulator(publicAPI, model) {
   // Set our className
   model.classHierarchy.push('vtkLineManipulator');
 
-  publicAPI.handleEvent = (callData, glRenderWindow) =>
-    projectDisplayToLine(
+  publicAPI.handleEvent = (callData, glRenderWindow) => ({
+    worldCoords: projectDisplayToLine(
```

Comment:
This is an API-breaking change, and so the commit must indicate as such with "BREAKING CHANGE:"  in the commit body. Additionally, this must be documented in the BREAKING_CHANGES.md doc.